packages/blocks/src/__internal__/utils/query.ts

```diff
@@ -29,6 +29,9 @@ export function getBlockById<T extends ElementTagName>(
   return container.querySelector<T>(`[${ATTR}="${id}"]` as T);
 }
 
+/*
+Be cautious!! If BlockSuite is embedded in other container, MouseEvent.clientX, MouseEvent.clientY will change
```

Comment:
I think e.raw.clientX Y is always what it is on the screen.

Reply:
But this value can't be compared with the staring position since they're not inside the same coord system.